src/main/java/com/teamsfinder/userwriteservice/user/dto/UserDto.java

```diff
@@ -0,0 +1,8 @@
+package com.teamsfinder.userwriteservice.user.dto;
+
+import com.teamsfinder.userwriteservice.tag.dto.TagDto;
+
+import java.util.List;
+import java.util.UUID;
+
+public record UserDto(Long id, String keyCloakId, String accountType, String githubProfileUrl, String profilePictureUrl, boolean blocked, List<TagDto> tags){ }
```

Comment:
W EditUserDto masz dodane adnotacje @NotNull, @NotEmpty itp. czemu tutaj nie?

Reply:
Ale to jest tylko zwracane, nigdzie tego nie przyjmuje, wiec po co walidacja?

Reply:
To może lepiej nazywać UserResponse ? Bo wtedy wiadomo, że to idzie w odpowiedzi a sama nazwa UserDto nie wiele mówi po za tym, że to dto